docs/pull_request_template.md

```diff
@@ -1,22 +1,44 @@
+<!--
```

Comment:
I added the template to the copyright check exception list, since it was cluttering the PRs (I think GitHub does not hide it, at least in edit mode)

Reply:
You are right, it will show in the PR description as GitHub